src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -1496,4 +1497,19 @@ public TopicChange publishTopic(TopicChange topicChange) {
   
   
 
+  @Override
+  public String translate(String text) {

```

Comment:
What determines the to and from languages for this translation call?

Reply:
pre existing config in the translator ...  setFromLanguage("en") setToLanguage("fr")